crypto/hash/hashfuncs.go

```diff
@@ -23,6 +23,8 @@ import (
 	//"github.com/minio/sha256-simd"
 	// "crypto/sha256" benchmark is at least 10% faster than "github.com/minio/sha256-simd"
 	"crypto/sha256"
+
+	"github.com/minio/blake2b-simd"
```

Comment:
I guess ```golang.org/x/crypto/blake2b``` may performance better

Reply:
"minio/blake2b-simd" benchmark is at least 3% faster on i7-4870HQ CPU @ 2.50GHz than "golang.org/x/crypto/blake2b" and supports more CPU instructions